src/main/resources/c2c-mvt.json

```diff
@@ -7,37 +7,46 @@
 			{
 				"encodings": ["UTF-8"],
 				"messageTypes": ["DMSControlRequest"],
-				"decoder": "usdot.fhwa.stol.c2c.c2c_mvt.decoders.JsonDecoder",
-				"parser": "usdot.fhwa.stol.c2c.c2c_mvt.parsers.NGTMDDJsonParser",
+				"decoder": "JsonDecoder",
+				"parser": "NGTMDDJsonParser",
+				"validator": "JsonValidator",
 				"schema": "ngTMDD/ngTMDD_Schema_v1.0.json"
 			},
-			"1.01" :
-			{
-			},
 			"1.1" :
 			{
-			},
-			"2.0" :
-			{
+				"encodings": ["UTF-8"],
+				"messageTypes": ["DMSControlRequest"],
+				"decoder": "JsonDecoder",
+				"parser": "NGTMDDJsonParser",
+				"validator": "JsonValidator",
+				"schema": "ngTMDD/ngTMDD_Schema_v1.1.json"
 			}
+
 		}
 	},
 	"TMDD":
 	{
 		"versions":
```

Comment:
versions could be an array rather than an object. Example: 
"versions":
		[
			
			{
                                 "versionNumber": "1.01",
				"encodings": ["UTF-8"],
				"messageTypes": ["DMSControlRequest"],
				"decoder": "JsonDecoder",
				"parser": "NGTMDDJsonParser",
				"validator": "JsonValidator",
				"schema": "ngTMDD/ngTMDD_Schema_v1.0.json"
			},
			
			{
                                 "versionNumber": "1.0",
				"encodings": ["UTF-8"],
				"messageTypes": ["DMSControlRequest"],
				"decoder": "JsonDecoder",
				"parser": "NGTMDDJsonParser",
				"validator": "JsonValidator",
				"schema": "ngTMDD/ngTMDD_Schema_v1.1.json"
			}

		]

Reply:
For both the standards and versions I intentionally used objects instead of arrays. The reason for this is that we need the list of available standards and versions and we look up things by standard name and version. This allows us to use Map.keySet() and store the standards and the version in Maps for quick lookup